src/OpenTelemetry.Api/Internal/SpanHelper.cs

```diff
@@ -19,15 +19,15 @@ namespace OpenTelemetry.Trace
     /// <summary>
     /// A collection of helper methods to be used when building spans.
     /// </summary>
-    public static class SpanHelper
+    internal static class SpanHelper
     {
         /// <summary>
         /// Helper method that populates span properties from http status code according
         /// to https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#status.
         /// </summary>
         /// <param name="httpStatusCode">Http status code.</param>
         /// <returns>Resolved span <see cref="Status"/> for the Http status code.</returns>
-        public static Status ResolveSpanStatusForHttpStatusCode(int httpStatusCode)
+        internal static Status ResolveSpanStatusForHttpStatusCode(int httpStatusCode)
```

Comment:
As class is anyway internal, this is okay to be left as public.

Reply:
Updated this to public.